classes/cache/Cache.php

```diff
@@ -290,8 +292,6 @@ public function deleteMulti(array $keyArray)
      *  E.g.: delete('*'); delete('my_prefix_*'); delete('my_key_name');.
      *
      * @param string $key
-     *
-     * @return array List of deleted keys
```

Comment:
Why remove this return? The method does return an array of keys

Reply:
Ok I get it, then this one should also return boolean to be consistent, you can change the return type in PHDoc

And modify the final return with `return true;`